railties/lib/rails/commands/rake_proxy.rb

```diff
@@ -0,0 +1,38 @@
+require 'rake'
+
+module Rails
+  module RakeProxy
+
+    RAKE_TASKS_HELP_MESSAGE = <<-EOT
+In addition to those, you can now run the rake tasks as rails commands:
```

Comment:
`now` doesn't make sense in the context of released software. When Rails 5 is 3 months old, is it still `now`? :smile:

We also want to go for as smooth a transition as possible, so the `rake` commands should just show up in the standard help output, I think.